CONTRIBUTING.md

```diff
@@ -181,13 +181,23 @@ follow the instructions in this section.
 
 ### Update the version, changelog, and notices
 1. Create a new branch for the version bump.
-1. Based on the unreleased content, determine the new version number and update
-   the [version](pkg/secrets/version.go) file.
+1. Based on the unreleased content, determine the new version number.
+1. Update this version in the following files:
+    1. [Version file](pkg/secrets/version.go)
+    1. [Chart version](helm/secrets-provider/Chart.yaml)
+    1. [Default deployed version](helm/secrets-provider/values.yaml)
+    
+1. Create a helm package by running the following command from the repo root: `pushd helm/secrets-provider/packages && helm package .. && popd`
+1. Add the resulting package file to the release PR.
```

Comment:
do we still need this step now that we pushed the .tgz to here https://github.com/cyberark/helm-charts/tree/master/docs

Looking here https://github.com/cyberark/conjur-oss-helm-chart/tree/master/conjur-oss they don't push the tgz alongside their helm charts